frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/active-health-checks.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { checkErrors } from '../../../integration-tests-cypress/support';
+import { MINUTE } from '../consts';
+
+describe('Test Popover behaviour for different active health check cases.', () => {
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    cy.install();
+    cy.visit('/ocs-dashboards/block-file');
+  });
+
+  after(() => {
+    checkErrors();
+    cy.logout();
+  });
+
+  const scaleDownAndOpenHC = (state: string, deployments: string[]) => {
+    const scaleTo = (replicas: number) =>
+      deployments.forEach((deployment) => {
+        cy.exec(`oc scale --replicas=${replicas} deploy ${deployment} -n openshift-storage`);
+      });
+    scaleTo(0);
+    cy.byTestID(`${state}-secondary-status`, { timeout: 2 * MINUTE }).should('be.visible');
+    cy.byTestID('Storage Cluster-health-item')
```

Comment:
```suggestion
    cy.byTestID('storage-cluster-health-item')
```
Odd way to name the data-tests. Can you change all of them to the pattern we usually follow ?